Dockerfile

```diff
@@ -8,6 +8,6 @@ LABEL com.github.actions.color="red"
 
 WORKDIR /app
 COPY . /app
-RUN shards build --production
+RUN shards build --production --release -Dpreview_mt
```

Comment:
Ok for `-Dpreview_mt`.

But I think we got rid of `--release` flag in all ameba scripts because it takes much more time for building. And the profit in running the binary in release mode is really small because Ameba has relatively small set of rules and perform really quickly on the average crystal project.

Reply:
But this container is built once and deployed many times, right? So the small delay in building the container can in aggregate save a lot of cpu cycles for all thousands/millions of CI jobs running the one container?

Reply:
Okay, makes sense.